Geometry/HcalCommonData/src/HcalDDDRecConstants.cc

```diff
@@ -226,7 +227,7 @@ HcalDDDRecConstants::getHFCellParameters() const {
   if (maxDepth[2] > 0) {
     for (unsigned int k=0; k<nEta; ++k) {
       int ieta = iEtaMin[2] + k;
-      int dphi = (int)(0.001 + hcons.getPhiTableHF()[k]/(5.0*CLHEP::deg));
+      int dphi = (int)(0.001 + hcons.getPhiTableHF()[k]/(5._deg));
```

Comment:
If you feel like doing more fixing, 0.001 and 72 could be replaced with constants, and the casts could be changed into static casts. Or they can be left to be fixed later.

Reply:
I prefer to fix these later. Let the bug fix be done first.